docs/markdown/authoritative/backend-ldap.md

```diff
@@ -97,16 +113,24 @@ In case the used LDAP client library doesn't support LDAP URIs as connection par
 ## `ldap-filter-lookup`
 (default "(:target:)" ) : LDAP filter for limiting IP or name lookups, e.g. (&(:target:)(active=yes)) for returning only entries whose attribute "active" is set to "yes".
 
+## `ldap-dnssec`
+(default: "no") : If set to "yes" DNSSEC support will be enabled in the backend.
+
+## `ldap-metadata-searchdn`
+(default: "") : For DNSSEC certain extra information must be stored that should not be found under the DNS data root. This is because PowerDNS must have write access to this subtree to, for example, create the zone keys. This setting point to the root DN under which the metadata can be stored. It is required if `ldap-dnssec` is set to "yes".
+
+## `ldap-metadata-searchfilter`
+(default: "(&(objectClass=organizationalUnit)(ou=:domain:))") : The search filter to use to find the DN under which the metadata for a zone is. The special string ":domain:" will be replace by the zone name in the search filter.
+
 # Master Mode
 
 Schema update
 -------------
 
-First off adding master support to the LDAP backend needs
-a schema update. This is required as some metadata must
-be stored by PowerDNS, such as the last successful transfer
-to slaves. The new schema is available in
-schema/pdns-domaininfo.schema.
+First off adding master support to the LDAP backend requires
```

Comment:
I have to trust you on this one, English is not my first language. Then it seems better this way, it'll be part of the next push, thanks!